contracts/utils/contracts/src/Authorizable.sol

```diff
@@ -43,19 +43,7 @@ contract Authorizable is
         external
         onlyOwner
     {
```

Comment:
Nit - I see there isn't actually a constructor in `Authorizable`. Maybe we could add one, which for now would just call `Ownable()`. Just to abstract it away from contracts that inherit this.

Reply:
Makes sense. I don't think it will make a difference at the bytecode level, but good to be explicit.